openedx/frontend-app-authoring
# Menu option to delete a component + small fixes [FC-0062]

## Description

* Implements https://github.com/openedx/frontend-app-authoring/issues/1391 to allow deleting components
   - ![Screenshot 1](https://github.com/user-attachments/assets/30414ee3-284a-4884-bc1b-f3c99b8e7e86)
   - ![Screenshot 2](https://github.com/user-attachments/assets/795e3ca8-755a-47a1-b91c-71c0161ad8c8)



## Testing instructions

Try deleting a component. It's easy :)

The other things should be obvious how to test them.

Private ref: [FAL-3905](https://tasks.opencraft.com/browse/FAL-3905)
